packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -250,9 +258,12 @@ class _GoogleMapState extends State<GoogleMap> {
 
   @override
   Widget build(BuildContext context) {
-    return GoogleMapsFlutterPlatform.instance.buildView(
+    return GoogleMapsFlutterPlatform.instance.buildViewWithTextDirection(
       _mapId,
       onPlatformViewCreated,
+      textDirection: widget.textDirection ??
+          Directionality.maybeOf(context) ??
```

Comment:
This may return `null` if there is no `Directionality` ancestor. This was the case with the tests, so I also added a default `TextDirection.ltr`.